src/status_im/contexts/profile/settings/view.cljs

```diff
@@ -79,7 +79,7 @@
                                                                      profile)}}])}]}]]
      [rn/flat-list
       {:header                          [settings.header/view {:scroll-y scroll-y}]
-       :data                            (settings.items/items (boolean (:mnemonic profile)))
+       :data                            (settings.items/items (boolean (seq (:mnemonic profile))))
```

Comment:
Fixes issue on showing that the user needs to backup their key, while they should not